README.md

```diff
@@ -58,8 +58,114 @@ helm:
   plugins:
 ```
 
-## CLI Configuration
+This version of ``helm`` plugin loops over the list of all helm chart folders of the operations repo and install in a sequential manner.
 
+However if a helm chart folder name is passed in ``BITOPS_ENVIRONMENT_HELM_SUBDIRECTORY`` variable in the docker run, this plugin will only install that specific helm chart. Below is the sample docker run showing how to install only one helm chart.
+
+```
+docker run --rm --name bitops \
+-e AWS_ACCESS_KEY_ID="${BITOPS_AWS_ACCESS_KEY_ID}" \
+-e AWS_SECRET_ACCESS_KEY="${BITOPS_AWS_SECRET_ACCESS_KEY}" \
+-e AWS_DEFAULT_REGION="${BITOPS_AWS_DEFAULT_REGION}" \
+-e BITOPS_ENVIRONMENT="${ENVIRONMENT}" \
+-e BITOPS_ENVIRONMENT_HELM_SUBDIRECTORY="ingress-nginx"  \
+-v $(pwd):/opt/bitops_deployment \
+bitops-core:latest
+
+```
+
+## CLI and options configuration of helm ``bitops.schema.yaml``
+
+### Helm BitOps Schema
```

Comment:
@shyamrayaprolu Can we just link to the schema with a note instead of putting the full contents here?

We'd have to maintain 2 copies otherwise.

Reply:
@mickmcgrath13 I just updated the schema with github link.